src/Types.jl

```diff
@@ -231,6 +231,8 @@ struct EnvCache
     uuids::Dict{String,Vector{UUID}}
     paths::Dict{UUID,Vector{String}}
 
+    preview::Ref{Bool}
```

Comment:
I should have mentioned this the first time around but it might be better to just make this strict mutable. There’s no real reason for it to be immutable.